README.md

```diff
@@ -789,10 +789,8 @@ test=> select session_user, current_user, user, current_role;
 
 ## NOTES
 
-### Version 4.0.0-rc1
-
-- Use of GUCs with `whitelist` have been removed from the codebase. Please use
-  the newer `allowlist` GUCs.
+### Version 4.0.0
```

Comment:
Super nitpicky, but do you mind putting whitespace between the subheading and the bulletpoint?